Src/Witsml/Data/Curves/DateTimeIndex.cs

```diff
@@ -5,24 +5,22 @@ namespace Witsml.Data.Curves
 {
     public class DateTimeIndex : Index
     {
-        private readonly DateTime dateTime;
+        private readonly DateTime _dateTime;
         public const string IsoPattern = "yyyy-MM-ddTHH:mm:ss.fffZ";
         public const string NullValue = "1900-01-01T00:00:00.000Z";
 
         public DateTimeIndex(DateTime dateTime)
         {
-            this.dateTime = dateTime;
+            this._dateTime = dateTime;
         }
 
-        public DateTime Value => dateTime;
+        public DateTime Value => _dateTime;
 
         public static DateTimeIndex FromString(string dateString)
         {
-            if (DateTime.TryParseExact(dateString, IsoPattern, null, DateTimeStyles.None, out var parsedDateTime))
-            {
-                return new DateTimeIndex(parsedDateTime);
-            }
-            throw new Exception($"Date format not recognized: {dateString}");
+            return (DateTime.TryParseExact(dateString, IsoPattern, null, DateTimeStyles.None, out var parsedDateTime))
```

Comment:
Unnecessary parentheses, do we ignore IDE0047?

Reply:
No I don't think so. 
It seems to me other warnings take precedence, when they are corrected IDE0047 will probably signal